template/static/bulmacard.css

```diff
@@ -0,0 +1,29 @@
+.card {
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+    background-size: cover;
+    width: 300px;
+    height: 300px;
+  }
+
+  .card {
+    margin: 0 auto; /* Added */
+    float: none; /* Added */
+    margin-bottom: 10px; /* Added */
+}
+
+.card {
+    position: relative;
+    padding-bottom: 10px;
+  }
+
+  .card {
+    margin-left: 500px;
+    margin-right: 200px;
+    width: 20;
+    border-radius: 30px;
+    position: relative;
+    display: block;
+    margin-top: 100px;
+    overflow: visible;
+  }
```

Comment:
This entire file has the style only for card. Why it is like this?

Reply:
I'm too poor on the CSS and HTML

Reply:
And also, my whole focus on the result only.that's why I couldn't on the styles.

Reply:
Do you understand what is wrong in that file?

Reply:
yes, I understand. that is, writing the styles for only **card**

Reply:
Writing styles only for card class is fine. But you wrote it multiple times. That's what the issue. You can put all the styles related to card in one placr

Reply:
oh..okay, I understand.